src/css/blocks/stars.css

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  border-radius: 0.5px;
+  background: url(../../images/star.svg) center no-repeat;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star,
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+2),
+.stars--3 .stars__star:nth-child(-n+3),
+.stars--4 .stars__star:not(:last-child) {
+  background: url(../../images/star-active.svg) center no-repeat;
```

Comment:
you should change only background-image here
`center` and `no-repeat` are redundant, they don't change

Reply:
Done.